samples/NASL/function.inc

```diff
@@ -0,0 +1,11 @@
+// Sample function
```

Comment:
Most of the sample files seem to be very simple. We prefer samples from "real" applications as they will be used to train the Bayesian classifier (i.e., they should be representative of actual usage).

Reply:
The latest commit removes these trivial samples and adds a bunch of real-life ones.